packages/utils/src/lib/logger.unit.test.ts

```diff
@@ -32,29 +26,20 @@ describe('formatCommand', () => {
     ['success' as const, ansis.green],
     ['failure' as const, ansis.red],
   ])(`should format command status %s explicitly`, (status, color) => {
-    expect(formatCommand('npx eslint . --format=json', {}, status)).toContain(
-      `${color('$')}`,
-    );
-  });
-
-  it('should include cwd prefix when cwd is provided and different from process.cwd()', () => {
-    const mockCwd = path.join(originalCwd, 'src');
-    vi.spyOn(path, 'relative').mockReturnValue('src');
-
-    expect(formatCommand('npx -v', { cwd: mockCwd })).toStartWith(
-      `${ansis.blue('src')} `,
-    );
+    expect(
+      formatCommandStatus('npx eslint . --format=json', {}, status),
+    ).toContain(`${color('$')}`);
   });
```

Comment:
Why did you delete this test case? It only requires a small adjustment to run without mocking `path.relative`. Without it, there's no coverage for the CWD prefix.

```ts
  it('should include cwd prefix when cwd is provided and different from process.cwd()', () => {
    expect(
      formatCommandStatus('npx -v', { cwd: path.join(process.cwd(), 'src') }),
    ).toStartWith(`${ansis.blue('src')} `);
  });
```

Reply:
reverted